Admin.js

```diff
@@ -34,78 +35,99 @@ var Admin = {
         Admin.setup_sticky_elements(subject);
         Admin.setup_readmore_elements(subject);
 
-//        Admin.setup_list_modal(subject);
+        //        Admin.setup_list_modal(subject);
+
     },
-    setup_list_modal: function(modal) {
-        Admin.log('[core|setup_list_modal] configure modal on', modal);
-        // this will force relation modal to open list of entity in a wider modal
-        // to improve readability
-        jQuery('div.modal-dialog', modal).css({
-            width:  '90%', //choose your width
-            height: '85%',
-            padding: 0
+    "setup_list_modal" (modal) {
+
+        Admin.log("[core|setup_list_modal] configure modal on", modal);
+        // This will force relation modal to open list of entity in a wider modal
+        // To improve readability
+        jQuery("div.modal-dialog", modal).css({
+            "width": "90%", // Choose your width
+            "height": "85%",
+            "padding": 0
         });
-        jQuery('div.modal-content', modal).css({
-            'border-radius':'0',
-            height:   '100%',
-            padding: 0
+        jQuery("div.modal-content", modal).css({
+            "border-radius": "0",
+            "height": "100%",
+            "padding": 0
         });
-        jQuery('.modal-body', modal).css({
-            width:    'auto',
-            height:   '90%',
-            padding: 15,
-            overflow: 'auto'
+        jQuery(".modal-body", modal).css({
+            "width": "auto",
+            "height": "90%",
+            "padding": 15,
+            "overflow": "auto"
         });
 
-        jQuery(modal).trigger('sonata-admin-setup-list-modal');
+        jQuery(modal).trigger("sonata-admin-setup-list-modal");
+
     },
-    setup_select2: function(subject) {
+    "setup_select2" (subject) {
+
         if (window.SONATA_CONFIG && window.SONATA_CONFIG.USE_SELECT2) {
-            Admin.log('[core|setup_select2] configure Select2 on', subject);
 
-            jQuery('select:not([data-sonata-select2="false"])', subject).each(function() {
-                var select            = jQuery(this);
-                var allowClearEnabled = false;
-                var popover           = select.data('popover');
+            Admin.log("[core|setup_select2] configure Select2 on", subject);
+
+            jQuery("select:not([data-sonata-select2=\"false\"])", subject).each(function () {
```

Comment:
- eslint: Line 72 exceeds the maximum line length of 80.
- eslint: Unexpected unnamed function.